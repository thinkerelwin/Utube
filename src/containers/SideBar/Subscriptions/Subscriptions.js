import React from 'react';
import { Subscription } from './Subscription/Subscription';
import { SideBarHeader } from '../SideBarHeader/SideBarHeader';

export const Subscriptions = () => (
  <>
    <SideBarHeader title="Subscriptions" />
    <Subscription label="MusicChannel" broadcasting />
    <Subscription label="freeCodeCamp" amountNewVideos={10} />
    <Subscription label="Udemy" amountNewVideos={4} />
    <Subscription label="Frontend Masters" amountNewVideos={23} />
    <Subscription label="Academnd" amountNewVideos={16} />
  </>
);
