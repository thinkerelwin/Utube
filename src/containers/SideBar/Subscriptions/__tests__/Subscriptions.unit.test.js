import React from 'react';
import { shallow } from 'enzyme';
import { Subscriptions } from '../Subscriptions';

test('renders Subscriptions', () => {
  const wrapper = shallow(<Subscriptions />);
  expect(wrapper).toMatchSnapshot();
});
