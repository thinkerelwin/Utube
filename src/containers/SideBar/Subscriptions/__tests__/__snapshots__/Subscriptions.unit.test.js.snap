// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Subscriptions 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="MusicChannel"
  />
  <Subscription
    amountNewVideos={10}
    label="Coursea"
  />
  <Subscription
    amountNewVideos={23}
    label="TEDx Talks"
  />
  <Subscription
    amountNewVideos={4}
    label="Stanford iOS"
  />
  <Subscription
    amountNewVideos={114}
    label="Udacity"
  />
</Fragment>
`;
